pkg/controller/nodelifecycle/node_lifecycle_controller_test.go

```diff
@@ -3703,7 +3703,7 @@ func TestReconcileNodeLabels(t *testing.T) {
 		if err := nodeController.syncNodeStore(fakeNodeHandler); err != nil {
 			t.Fatalf("unexpected error: %v", err)
 		}
-		nodeController.reconcileNodeLabels(test.Node.Name)
+		nodeController.reconcileNodeLabels(context.TODO(), test.Node.Name)
```

Comment:
Please use ktesting in unit tests. See for example https://github.com/kubernetes/kubernetes/pull/113464/files#diff-20caa98b2e20ceb93416bcd3d72559d7aba0c0a6f067032ec162d30540a30196

Reply:
Done. But I only update the `TestReconcileNodeLabels` function, but I saw other test functions in this file do not use ktesting either. Do I need to update them in this pr? @pohly

Reply:
Yes, please remove all `context.TODO` from this file by creating a context with `NewTestContext`.

Reply:
@pohly All `context.TODO` in the test functions of this file are replaced by `NewTestContext`